src/templates/views/index.pug

```diff
@@ -64,8 +64,8 @@ body
           //-   include ../pages/updates
           //- #tabs-news.tab-pane
           //-   include ../pages/news
-          #tabs-notifications.tab-pane
-            include ../pages/notifications
+          //- #tabs-notifications.tab-pane
```

Comment:
Why are we removing notifications tab? Eventually we will bring back this section for them to opt-in phone number. The notifications tab content should be updated instead

Reply:
I received the feedback from Erik about notifications tab since it's not working right now. What should the tab content be changed to? Just to say "Notifications are in the works"?